compiler/rustc_hir_typeck/src/lib.rs

```diff
@@ -47,6 +47,7 @@ mod writeback;
 pub use diverges::Diverges;
 pub use expectation::Expectation;
 pub use fn_ctxt::*;
+use hir::def_id::CRATE_DEF_ID;
```

Comment:
This belongs with imports from other crates.
```suggestion
use rustc_hir::def_id::CRATE_DEF_ID;
```